### ISSUE TITLE
Feature request: more (custom) events

### ISSUE THREAD

**Issue by NemWar001** [andrewdsmith/edxlc#3]
Hi,

nice tool, would be great if there is a way to add more events (like attacked, ship docked or something like that).

**Comment by andrewdsmith** [comment]
Are you suggesting more modes in addition to night vision and hardpoints deployed? So modes for under attack, docked, etc.?

**Comment by NemWar001** [comment]
Yes, under attack, docked, overheating... i'm note sure what options are possible... but i think some more would be nice...

**Comment by andrewdsmith** [comment]
The interesting problem to solve here is the precedence order. So if you have hardpoints out, night vision on, and you're being attacked, which setting wins? I guess I could add an order value so people could choose.

Anyway, I'm super busy on other projects right now so I'm unlikely to get to this any time soon, sorry. I'd welcome pull requests should anyone want to take some or all of this on.
